tests-integration/reference-model/src/Morphir/Reference/Model/BooksAndRecords.elm

```diff
@@ -25,38 +25,34 @@ type alias Quantity =
 
 
 {- Identifies a structure that can be associated to a persistance entity -}
-
-
-type alias Deal =
-    { product : ProductID
-    , price : Price
-    , quantity : Quantity
-    }
-
-
-
-{- These define the requests that can be made of this service -}
-
-
-type DealCmd
-    = OpenDeal ProductID Price Quantity
-    | CloseDeal ProductID
-
-
-
-{- These define the responses that would result from requests -}
-
-
-type DealEvent
-    = DealOpened ProductID Price Quantity
-    | DealClosed ProductID
-    | InvalidQuantity Quantity
-    | InvalidPrice Price
-    | DuplicateDeal ProductID
-    | DealNotFound ProductID
-
-
-
+--
+--type alias Deal =
+--    { product : ProductID
+--    , price : Price
+--    , quantity : Quantity
+--    }
+--
+--
+--
+--{- These define the requests that can be made of this service -}
+--
+--
+--type DealCmd
+--    = OpenDeal ProductID Price Quantity
+--    | CloseDeal ProductID
+--
+--
+--
+--{- These define the responses that would result from requests -}
+--
+--
+--type DealEvent
+--    = DealOpened ProductID Price Quantity
+--    | DealClosed ProductID
+--    | InvalidQuantity Quantity
+--    | InvalidPrice Price
+--    | DuplicateDeal ProductID
+--    | DealNotFound ProductID
```

Comment:
Did you submit this by accident? I don't think this should be commented out.

Reply:
This has been corrected now.